src/extensions/extensionsHelper.ts

```diff
@@ -178,3 +180,19 @@ export async function ensureExtensionsApiEnabled(options: any): Promise<void> {
     options.markdown
   );
 }
+
+/**
+ * Displays list of all official extensions and prompt user to select one.
+ * @param message The prompt message to display
+ * @returns Promise that resolves to the extension name (e.g. storage-resize-images)
+ */
+export async function promptForOfficialExtension(message: string): Promise<string> {
+  const officialExts = await getExtensionRegistry();
+  return await promptOnce({
+    name: "input",
+    type: "list",
+    message,
+    choices: convertOfficialExtensionsToList(officialExts),
+    pageSize: _.size(officialExts),
+  });
+}
```

Comment:
Why does this need to be in a "higher level" helpers file? This looks pretty specific to the install flow, right? Should this stay in `ext-install.ts`?

Reply:
Disagree that this is install specific - I could see us adding this 'list all official extensions' functionality to ext:info, and possibly other places in the future.

Reply:
+1